### PR TITLE
Tester branch

### PR DESCRIPTION
**Pull request by JeremiahHerring**
can you review my molest branch for me?

### CODE REVIEW

File: input.txt [hunk 1]
```diff
@@ -1 +1 @@
-while (fahr <= upper) a = 23.00;
\ No newline at end of file
+<=
```
Comment (noahbei) [inline input.txt:1]: Hm, interesting choice here

File: Lexer FSM/RealLexer.py [hunk 2]
```diff
@@ -1,19 +1,23 @@
+from Separator import separators
+from Operator import operators
+
 class RealLexer:
     def __init__(self):
         self.states = {'1', '2', '3', '4'}
-        self.alphabet = {'digit', '.'}
+        self.alphabet = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '.'}
```
Comment (noahbei) [inline Lexer FSM/RealLexer.py:7]: digit :)

**Review by noahbei — COMMENTED** [review]
cool beans

File: Lexer FSM/RealLexer.py (hunk 2 above)
Comment (JeremiahHerring) [reply Lexer FSM/RealLexer.py:7]: AAHHHHHH

File: input.txt (hunk 1 above)
Comment (JeremiahHerring) [reply input.txt:1]: WHAT DO YOU MEAN

**Pull request merged**